app/pods/contests/contest/index/route.js

```diff
@@ -26,6 +29,14 @@ export default class IndexRoute extends Route {
     return this.modelFor('contests.contest')
   }
 
+  afterModel(model) {
+    this.metrics.trackEvent({
+      event: 'Contest View',
+      title: model.contest.name,
+      page: 'course'
```

Comment:
Page should be url of current page here, in case of frontend tracking

Reply:
done !